is.js

```diff
@@ -586,6 +586,19 @@
     var root = freeGlobal || freeSelf || thisGlobal || Function('return this')();
 
     var previousIs = root.is;
+  
+    // is current runtime node.js?
+    // parameter is optional
+    is.node = function(range) {
+        if (toString.call(typeof process !== 'undefined' ? process : 0) === '[object process]'){
```

Comment:
feel free to change it to whatever style you think is best

i copied the code from https://github.com/parshap/check-node-version

im not gonna make tiny changes every 10 min
you are welcome